htdocs/core/class/html.form.class.php

```diff
@@ -1511,8 +1511,8 @@ public function select_remises($selected, $htmlname, $filter, $socid, $maxvalue
 		global $langs, $conf;
 
 		// On recherche les remises
-		$sql = "SELECT re.rowid, re.amount_ht, re.amount_tva, re.amount_ttc,";
-		$sql .= " re.description, re.fk_facture_source";
+		$sql = "SELECT re.rowid, re.amount_ht, re.amount_tva, re.amount_ttc, re.multicurrency_amount_ttc,";
+		$sql .= " re.description, re.fk_facture_sourcere, re.fk_invoice_supplier_source";
```

Comment:
Field fk_facture_sourcere seems wrong

Reply:
You are right. some typo error, as I am new to Github.